doc/source/ndl.rst

```diff
@@ -1,8 +1,30 @@
+Naive Discriminative Learning
```

Comment:
we  should add a quick note about equilibruum solutions to this section.

Reply:
I opened an issue for that (#133) to resolve it later